main.go

```diff
@@ -130,14 +137,24 @@ func mainerr() error {
 	// Then, add the user's docker flags.
 	allDockerArgs = append(allDockerArgs, dockerFlags...)
 
-	// Add "--" to stop all docker flags, plus the specified image.
-	allDockerArgs = append(allDockerArgs, "--", image)
+	// Add "--" to stop all docker flags
```

Comment:
also mention that this is not available in docker-compose, and link that bug you opened

Reply:
Thanks - totally meant to do just that.